csi/Cargo.toml

```diff
@@ -1,10 +1,11 @@
 [package]
 description = "gRPC bindings to the CSI spec"
+name = "bpfman-csi"
+version = "1.8.0"
```

Comment:
Does this need `documentation.workspace = true`?

Reply:
I think that's outdated and the current version doesn't have that anymore.

Reply:
You lost me. For all the other `Cargo.toml` files, `documentation.workspace = true` was added, but `csi/Cargo.toml` was the only one without it. I don't follow all the changes, just watching the pattern and noticed an anomaly. It's fine if it doesn't need it, just curious.

Reply:
So I intentionally left it out here because technically these bindings are independent of the bpfman project, i.e the crate could be used by anyone as rust CSI protobuf bindings.

Reply:
Good enough for me, just wanted to make sure it wasn't an oversight.